spy/bin/webpack.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
Why a bin folder ?
webpack.config.js is supposed to be in a src folder

Reply:
Do not push the bin folder
For example this file already exists in the repository